analyses/cell-type-ewings/exploratory_analysis/annotation_notebooks/SCPCS000492/SCPCL000824_singler-classification.Rmd

```diff
@@ -0,0 +1,539 @@
+  "updated_singler_blueprint_classification_lumped"
+) |> 
+  purrr::set_names("Tumor only", "Original", "Original and Blueprint")
+
+# compare blueprint to tumor, original and original + blueprint
+plot_jaccard(classification_df,
+             annotation_column = "blueprint_tumor_classification_lumped",
+             methods_to_compare = methods,
+             column_title = "Blueprint",
+             legend_match = "Original") |> 
+  ComplexHeatmap::draw(
+    heatmap_legend_side = "right",
+    # add a margin to the heatmap so labels don't get cut off
+    padding = unit(c(2, 20, 2, 2), "mm")
+  )
+
+# compare original to tumor, original + blueprint
+subset_methods <- c(
+  "tumor_only_classification_lumped",
+  "updated_singler_blueprint_classification_lumped"
+) |> 
+  purrr::set_names("Tumor only", "Original and Blueprint")
+
+
+plot_jaccard(classification_df,
+             annotation_column = "updated_singler_classification_lumped",
+             methods_to_compare = subset_methods,
+             column_title = "Original",
+             legend_match = "Original")
+
+# compare tumor to original and blueprint
+plot_jaccard(classification_df,
+             annotation_column = "tumor_only_classification_lumped",
+             methods_to_compare = c("Original and Blueprint" = "updated_singler_blueprint_classification_lumped"),
+             column_title = "Tumor only",
+             legend_match = "Tumor only")
+  
+```
+
+A few things that we see in these plots: 
+
+- There seems to be mostly consensus across all four methods on which cells are tumor cells, which is consistent with the upset plot. 
+- When only using Blueprint, a lot of cells are categorized as "Normal". 
+This means they match the non-tumor cells identified in `SCPCL000822` more than they match the normal cell types in the `Blueprint` reference. 
+Additionally, with only Blueprint, there is a group of cells that are called as tumor but actually appear to match endothelial cells present in `SCPCL000822`.
+- The cells that aren't called as either Tumor or Normal in the Blueprint only reference mostly match up to `erythrocyte` in the reference that just uses `SCPCL000822` (original). 
+These same cells match to their corresponding Blueprint cell type when combining `SCPCL000822` and Blueprint (e.g., fibroblasts, neurons, and epithelial cells are identifed in both). 
+- When comparing using just `SCPCL000822` (original) to `SCPCL000822` and Blueprint (original and blueprint), we see an overlap in endothelial cells and chrondrocytes. 
+We also see that for the most part the top cell types identified are the same except when using blueprint you see the addition of epithelial cells. 
+- Using tumor only appears to label some cells as tumor that are labeled as normal with other methods. 
+For example, comparing tumor only to both original and original and blueprint shows that tumor cells are classified as endothelial cells using either of the other two methods.  
```

Comment:
Do you mean whether or not the tumor cells are correct? Or if the calls for them being endothelial cells are correct? I think multiple methods calling them endothelial cells (SingleR, CellAssign, and then using our custom refs here) makes me trust them to be endothelial over tumor?

Reply:
I do mean whether calling them tumor or not is correct.

My question here is really whether the "tumor only" ref is catching extra true tumor cells or if those cells were being correctly assigned by the "original" method.  From the UMAPs it did look like the "tumor only" was calling more cells outside the lower group which I would have expected were false positives, but I wasn't sure if that was a reasonable conclusion. I don't think it is something we have to know at this point, but I wasn't sure if you had a sense.

Reply:
I don't know for sure, but one way to look at that is specifically pull out those cells and see if they are called tumor by marker gene methods. I think that's the only way I would really trust that they are tumor at this point. My sense is that they are not since on the marker gene UMAPs they are also considered normal cells.